libuavcan_drivers/stm32/driver/src/uc_stm32_can.cpp

```diff
@@ -16,6 +16,9 @@
 # include <arch/board/board.h>
 #elif UAVCAN_STM32_BAREMETAL
 #include <chip.h>
+#elif UAVCAN_STM32_FREERTOS
+#include <stm32f4xx.h>
+#include <cmsis_os.h>
```

Comment:
These includes are redundant, see `internal.hpp` (yes, some of the includes above them are also redundant, that's a separate problem).